configs/test/gce/windows-init.ps1

```diff
@@ -266,31 +281,40 @@ $deploymentBucket = $webClient.DownloadString('http://metadata.google.internal/c
 # Download ClusterFuzz source.
 rm c:\clusterfuzz -Recurse -Force
 $fileName = "$tmp\clusterfuzz.zip"
-gsutil cp gs://$deploymentBucket/windows.zip $fileName
+gsutil cp gs://$deploymentBucket/windows-3.zip $fileName
 unzip $fileName
 
 # Resize partition to max available size.
 $MaxSize = (Get-PartitionSupportedSize -DriveLetter c).sizeMax
 Resize-Partition -DriveLetter c -Size $MaxSize
 
-Set-Content $scriptExecutedFilePath "done"
+Set-Content $packageSetupFilePath " "
 
 # Run autologin so that the hostname changes at reboot.
 c:\autologin.bat
 
-Write-Host "Restart"
+Write-Host "Restarting"
 
 Restart-Computer -Force
 exit
 
-} # !(Test-Path ($scriptExecutedFilePath))
+} # !(Test-Path ($packageSetupFilePath))
 
-Set-Content $scriptExecutedFilePath "skipped"
+Set-Content $packageSetupFilePath "Skipped package install"
 
 # Schedule chkdsk on every reboot.
 echo y | chkdsk C: /F /I /C
 
+# Install Pipfile dependencies
```

Comment:
Any reason why these two blocks are not inside packageSetupFilePath ? They should be done once i think , not everytime.

Reply:
This needs to happen after we download the source.

Reply:
Makes sense as we change clusterfuzz source! LGTM